pkg/kubectl/cmd/rollout/rollout_pause.go

```diff
@@ -125,7 +119,7 @@ func (o *PauseConfig) CompletePause(f *cmdutil.Factory, cmd *cobra.Command, out
 	return nil
 }
 
-func (o PauseConfig) RunPause() error {
+func (o PauseConfig) RunPause() {
```

Comment:
@kargakis I thought my original implementation does that (checkErr in the caller)